guide/src/class/numeric.md

```diff
@@ -171,7 +171,7 @@ impl Number {
     }
 
     fn __complex__<'py>(&self, py: Python<'py>) -> &'py PyComplex {
-        PyComplex::from_doubles(py, self.0 as f64, 0.0)
+        PyComplex::from_doubles_bound(py, self.0 as f64, 0.0).into_gil_ref()
     }
```

Comment:
Does it work already to have this implementation?

```rust
    fn __complex__<'py>(&self, py: Python<'py>) -> Bound<'py, PyComplex> {
         PyComplex::from_doubles_bound(py, self.0 as f64, 0.0)
     }
```

Reply:
Jup, this works already, nice 👍